web_client/views/configView.js

```diff
@@ -115,7 +115,11 @@ var ConfigView = View.extend({
             name: 'slideatlas',
             label: 'SlideAtlas',
             type: 'slideatlas'
-        }
+        }, {
+            name: 'geospatial',
```

Comment:
Can you name this so that it is obvious that it is an openlayers based geospatial viewer?  Eventually, a geojs geospatial viewer would also be nice.